### ISSUE TITLE
[FR] Filter processing

### ISSUE THREAD

**Issue by anayrat** [FreshRSS/FreshRSS#1368]
Add the possibility to perform action according to rules: 

Filter "Postgres" : 
  - Postgres in title or content

Perform : 
 - Mark as read
 - Assign tag "postgres"

**Comment by anayrat** [comment]
For information this functionnality is in tinyrss. It is missing for my usage :(

**Comment by Alkarex** [comment]
Currently, it is already possible to (manually) search for a string, and mark the results as read.

**Comment by SebCorbin** [comment]
Hello there, just moved from Feedly to FreshRSS after balancing out between TinyTinyRSS and FreshRSS. My motivation was that FreshRSS looks very nice and is mobile friendly.
Anyway: good work!

But filters are a feature that I miss too, I did not expect it to be only "filter articles list," but instead more than an "automatically forget articles containing X", maybe it should be named "blacklist" or something like that?

**Comment by Alkarex** [comment]
@SebCorbin See the related issue for hiding spoilers https://github.com/FreshRSS/FreshRSS/issues/1161
Currently, this is not in the FreshRSS core (can be done manually, or by an extension, or by a cron job with curl). However, I have just increased its priority to try to include it in the next version 1.8 of FreshRSS.

**Comment by Jucgshu** [comment]
@alkarex Sorry to ask: has it been included to FreshRSS? Thanks for your work as always :wink:

**Comment by Alkarex** [comment]
@Jucgshu No, not yet, but for the time being, extensions are relatively easy to do for making custom rules.

**Comment by Alkarex** [comment]
First implementation available in https://github.com/FreshRSS/FreshRSS/pull/2275

**Comment by albertopasqualetto** [comment]
Should close as already implemented

**Comment by Frenzie** [comment]
Alright.
